modules/apps/remote-app/remote-app-service/service.xml

```diff
@@ -19,8 +19,12 @@
 
 		<!-- Other fields -->
 
+		<column name="customElementCSSURLs" type="String" uad-nonanonymizable="true" />
+		<column name="customElementHTMLElementName" type="String" uad-nonanonymizable="true" />
+		<column name="customElementURLs" type="String" uad-nonanonymizable="true" />
+		<column name="iframeURL" type="String" uad-nonanonymizable="true" />
```

Comment:
we mentioned avoiding the use of iframe to decouple from the implementation... and use app or something like it

Reply:
Yeah, but then I discussed it with @jbalsas privately and came to the conclusion that if a remote app based on custom elements is of type `customElement`, then a remote app based on an iframe should be `iframe`.

In the end, the `type` defines the contract between Liferay and the URL asset, and it makes sense to define them as `iframe` and `customElement`, we thought.

In the future we may add other types like: `jsWidget`, `amdModule`, `browserModule`, ...

Reply:
👍